package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "NODE_ENV=test mocha src/server/test",
     "start": "node bin/server.js",
-    "prestart": "sequelize db:migrate"
+    "prestart": "sequelize db:migrate",
```

Comment:
@kwhinnery Are you hinting at this? https://github.com/kwhinnery/todomvc-plusplus/issues/5#issuecomment-242461944